src/Private/Console.ps1

```diff
@@ -0,0 +1,987 @@
+            }
+        }
+        elseif (! (Test-PodeServerIsEnabled)) {
+            Enable-PodeServerInternal
+            Show-PodeConsoleInfo -ShowTopSeparator
+        }
+    }
+
+    # Handle suspend/resume actions
+    if ($PodeContext.Server.AllowedActions.Suspend) {
+        if ((! $PodeContext.Server.Console.DisableTermination) -and (Test-PodeKeyPressed -Key $Key -Character $KeyBindings.Suspend)) {
+            Clear-PodeKeyPressed
+            if ($serverState -eq 'Suspended') {
+                Set-PodeResumeToken
+                Resume-PodeServerInternal -Timeout $PodeContext.Server.AllowedActions.Timeout.Resume
+            }
+            elseif ($serverState -eq 'Running') {
+                Set-PodeSuspendToken
+                Suspend-PodeServerInternal -Timeout $PodeContext.Server.AllowedActions.Timeout.Suspend
+            }
+        }
+        elseif ((Test-PodeCancellationTokenRequest -Type Resume) -and ($serverState -eq 'Suspended')) {
+            Resume-PodeServerInternal -Timeout $PodeContext.Server.AllowedActions.Timeout.Resume
+        }
+        elseif ((Test-PodeCancellationTokenRequest -Type Suspend) -and ($serverState -eq 'Running')) {
+            Suspend-PodeServerInternal -Timeout $PodeContext.Server.AllowedActions.Timeout.Suspend
+        }
+    }
+}
+
+<#
+.SYNOPSIS
+    Retrieves the default console settings for Pode.
+
+.DESCRIPTION
+    The `Get-PodeDefaultConsole` function returns a hashtable containing the default console configuration for Pode. This includes settings for termination, console input, output formatting, timestamps, and color themes, as well as key bindings for console navigation.
+
+.OUTPUTS
+    [hashtable]
+        A hashtable representing the default console settings, including termination behavior, display options, colors, and key bindings.
+
+.EXAMPLE
+    $consoleSettings = Get-PodeDefaultConsole
+    Write-Output $consoleSettings
+
+    This example retrieves the default console settings and displays them.
+
+.NOTES
+    This is an internal function and may change in future releases of Pode.
+#>
+function Get-PodeDefaultConsole {
+    return @{
+        DisableTermination  = $false    # Prevent Ctrl+C from terminating the server.
+        DisableConsoleInput = $false    # Disable all console input controls.
+        Quiet               = $false    # Suppress console output.
+        ClearHost           = $false    # Clear the console output at startup.
+        ShowOpenAPI         = $true     # Display OpenAPI information.
+        ShowEndpoints       = $true     # Display listening endpoints.
+        ShowHelp            = $false    # Show help instructions in the console.
+        ShowDivider         = $true     # Display dividers between sections.
+        DividerLength       = 75        # Length of dividers in the console.
+        ShowTimeStamp       = $true     # Display timestamp in the header.
+
+        Colors              = @{            # Customize console colors.
```

Comment:
`Colours` 😉

Reply:
but you don't pronounce the u :)

Reply:
and will be weird to use Colours when everywhere in powershell is Color